### ISSUE TITLE
Checkmark is a little off position

### ISSUE THREAD

**Issue by DeFaLT-dj** [wseagar/eight-dollars#79]
It really annoys me. I love the extension but there is more space between name and checkmark which is kinda annoying. Sad I mainly use it for Twitter Blue, so it doesn't have any text etc. for official checkmarks. but there is still a gap between them as well. Here is an example between extension active and not active.  
 
Extension deactive:
![image](https://user-images.githubusercontent.com/64617514/209206993-686d53e8-4e40-413f-bc6f-7706d9e8fe87.png)
 
Extension active: There are gaps at both sides of the checkmark
![image](https://user-images.githubusercontent.com/64617514/209207026-b5603d12-8a73-478d-abe8-164ee86b87f7.png)

Twitter Blue with extension:

![image](https://user-images.githubusercontent.com/64617514/209206082-9e8ac851-26c6-44a7-a331-4ea71b2c5bd6.png)

**Comment by AngelofVerdant** [comment]
Seriouslyy😂😂😂😂

**Comment by DeFaLT-dj** [comment]
Edit: It only happens when the text is activated, it is easier to identify blue subscribers with text active so it is still an issue imo.

**Comment by DeFaLT-dj** [comment]
> Seriouslyy😂😂😂😂

Lol yeah, this kind of little thinks really bugs me off. It looks asymmetric and really annoys me haha

**Comment by waltzaround** [comment]
As a designer, I feel ya - @DeFaLT-dj  id recommend having a play with the CSS in https://github.com/wseagar/eight-dollars/blob/main/script.js if it bugs you that much :P

**Comment by DeFaLT-dj** [comment]
> As a designer, I feel ya - @DeFaLT-dj  id recommend having a play with the CSS in https://github.com/wseagar/eight-dollars/blob/main/script.js if it bugs you that much :P

Yeah, I just disabled text for now since it only happens when it is active. The only downside is you sometimes can't tell Blue subs at first look since it is much more obvious with text added. But I can work with that for now :p
